From 77f54f2463b9cef1975887d667e456e34190b18f Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Wed, 17 Jan 2024 19:20:42 -0600
Subject: [PATCH] Don't check for system kdsoap-ws-discover-client

The system one is only built for qt6, so cmake will fail if it tries to use it.

---
 smb/CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/smb/CMakeLists.txt b/smb/CMakeLists.txt
index 2105ff3ba..06d9f9963 100644
--- a/smb/CMakeLists.txt
+++ b/smb/CMakeLists.txt
@@ -7,11 +7,7 @@ remove_definitions(
 )
 
 option(BUILD_KDSoapWSDiscoveryClient "Automatically build WSD client if a system one isn't found." ON)
-find_package(KDSoapWSDiscoveryClient QUIET)
 set(INTERNAL_WSDCLIENT ${BUILD_KDSoapWSDiscoveryClient})
-if(KDSoapWSDiscoveryClient_FOUND)
-    set(INTERNAL_WSDCLIENT OFF)
-endif()
 if(INTERNAL_WSDCLIENT)
     # Special internal version, mangled to be a STATIC lib.
     # This is only useful and necessary until the library has
-- 
2.43.0

